app/routes/career.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react'
+import JobLists from '~/components/job-lists';
+import { JobSkeletonCard } from '~/components/job-skeleton-card';
+
+
+const jobTitles = ["Frontend Developer", "Backend Developer", "UI/UX Designer", "Product Manager", "DevOps Engineer"];
+const locations = ["New York, NY", "San Francisco, CA", "Remote", "Austin, TX", "Seattle, WA"];
+const descriptions = [
+    "Develop and maintain web applications using React.",
+    "Design and implement backend services using Node.js.",
+    "Create user-friendly and visually appealing designs.",
+    "Manage product development from conception to launch.",
+    "Automate and streamline operations and processes."
+];
+const salaryRanges = ["$70,000 - $90,000", "$80,000 - $100,000", "$90,000 - $110,000", "$100,000 - $120,000"];
+const jobTypes = ["Full-time", "Part-time", "Remote", "Contract"];
+const jobCategories = ["Software Development", "Design", "Product Management", "Operations"];
+
+const getRandomElement = (arr: string[]) => arr[Math.floor(Math.random() * arr.length)];
+
+
+// types.ts
+export type Job = {
+    jobTitle: String;
+    location: String;
+    description: String;
+    salaryRange: String;
+    jobType: String;
+    jobCategory: String;
+    detailsLink: String;
+}
+
+
+const generateRandomJob = (index: number): Job => {
+    return {
+        jobTitle: getRandomElement(jobTitles),
+        location: getRandomElement(locations),
+        description: getRandomElement(descriptions),
+        salaryRange: getRandomElement(salaryRanges),
+        jobType: getRandomElement(jobTypes),
+        jobCategory: getRandomElement(jobCategories),
+        detailsLink: `/jobs/${index}`
+    };
+};
+
```

Comment:
You were told to use clean code, right? Why are these logics here? I only expected UI rendering logic in this file.

Reply:
ok, working in it sir

Reply:
@chumex412  but what do i do while most of the components need to display on the page are not available

Reply:
Move the data and logic to a .ts file outside the .tsx file and import it into the .tsx file

Reply:
@chumex412 have you niticed our issue? Can I be merged so that @shuaibu-shehu may use the pegination I have implemented?

Reply:
see here https://github.com/hngprojects/hng_boilerplate_remix/pull/194#issuecomment-2241349278

Reply:
@chumex412  in could not find the figma design anymore